api/client/amp.go

```diff
@@ -87,32 +88,59 @@ type AMP struct {
 	// Conn is the gRPC connection to amplifier
 	Conn *grpc.ClientConn
 
+	// Conn is the gRPC connection to adm-server
+	ConnAdmServer *grpc.ClientConn
+
 	// Log also implements the grpclog.Logger interface
 	Log Logger
 }
 
+// IsLocalhost return true if the server address is localhost
+func (a *AMP) IsLocalhost() bool {
+	if strings.Index(a.Configuration.ServerAddress, "127.0.0.1:") >= 0 || strings.Index(a.Configuration.ServerAddress, "localhost:") >= 0 {
+		return true
+	}
+	return false
+}
+
 // Connect to amplifier
 func (a *AMP) Connect() error {
 	conn, err := grpc.Dial(a.Configuration.ServerAddress,
 		grpc.WithInsecure(),
 		grpc.WithBlock(),
-		grpc.WithTimeout(time.Second))
+		grpc.WithTimeout(time.Second*3))
```

Comment:
previously it was 1 seconds, ok for local amp install. I got issue with remote aws install, so the time allowed for the grpc connection is 3 seconds, to be adjusted if needed, nbut it seems ok for now.
no?